diff --git a/kernel/policy/app_profile.c b/kernel/policy/app_profile.c
--- a/kernel/policy/app_profile.c
+++ b/kernel/policy/app_profile.c
@@ -1,5 +1,6 @@
 #include <linux/version.h>
+#include <linux/sched.h>
 #include <linux/capability.h>
 #include <linux/cred.h>
 #include <linux/err.h>
 #include <linux/fdtable.h>
 #include <linux/file.h>
 #include <linux/fs.h>
 #include <linux/pid.h>
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
 // https://github.com/torvalds/linux/commit/8703e8a465b1e9cadc3680b4b1248f5987e54518
 #include <linux/sched/user.h>
 #include <linux/sched/task.h>
 #endif
-#include <linux/sched.h>
 #include <linux/seccomp.h>
 #include <linux/thread_info.h>
 #include <linux/uidgid.h>
 #include <linux/syscalls.h>
 #include "objsec.h"
 #include <linux/spinlock.h>
 #include <linux/tty.h>
 #include <linux/security.h>

 #include "policy/allowlist.h"
 #include "policy/app_profile.h"
 #include "arch.h"
 #include "compat/kernel_compat.h"
 #include "klog.h" // IWYU pragma: keep
 #include "selinux/selinux.h"
 #include "infra/su_mount_ns.h"
 #ifdef CONFIG_KSU_TRACEPOINT_HOOK
 #include "hook/tp_marker.h"
 #endif
